app/components/DiffTable.tsx

```diff
@@ -20,23 +20,26 @@ const StyledTable = styled.table`
   }
 `;
 
-const format = (value) => {
-  if (typeof value === 'number') {
+const format = (value, type) => {
+  if (typeof value === 'number' && type === 'number') {
     return formatMoney(value);
   }
   if (typeof value === 'boolean') {
```

Comment:
I know it's late in the game concerning this component, but this file is quite large and has many conditions and doesn't have any comments. Obviously not needed for this ticket but something we should do in the future.

Reply:
Agreed, I'll create a small ticket for it 👍